react/components/EXPERIMENTAL_Table/hooks/useTableBulkActions.tsx

```diff
@@ -233,4 +233,26 @@ type hookReturn = {
   selectAllVisibleRows?: () => void
 }
 
+export type BulkedItem = {
+  id: number
```

Comment:
I think this should be a string... All of our graphql ids are string

Reply:
I'm using a prop called `unicityKey` to deal with which property makes each object unique.